cmd/api/src/database/datapipestatus.go

```diff
@@ -26,23 +26,23 @@ import (
 
 func (s *BloodhoundDB) SetDatapipeStatus(ctx context.Context, status model.DatapipeStatus, updateAnalysisTime bool) error {
 
-	updateSql := "UPDATE datapipe_status SET status = ?, updated_at = ?"
+	updateSql := "UPDATE datapipe_status SET status = ?, updated_at = ?,last_analysis_run_at = ?"
```

Comment:
I think there needs to be a space after the `,` or this SQL will fail
```suggestion
	updateSql := "UPDATE datapipe_status SET status = ?, updated_at = ?, last_analysis_run_at = ?"
```